vembrane/representations.py

```diff
@@ -209,14 +209,19 @@ def __init__(
         # an index operation.
         self._numbers = {
             kind: {
-                record.get("ID"): overwrite_number.get(record.get("ID"))
+                record.get("ID"): overwrite_number.get(kind, {}).get(record.get("ID"))
                 or record.get("Number")
                 for record in header.records
                 if record.type == kind
             }
             for kind in set(r.type for r in header.records)
         }
 
+        # always explicitly set "Number" for certain fields
+        # which get special pysam treatment:
+        # - `FORMAT["GT"]` is always parsed as a list of integer values
```

Comment:
Does it make sense to reference the respective pysam code, here? For quicker reference, should this need adjustments or additions in the future?

Reply:
Not sure where in the code this happens, but there are several issues one could mention, such as: https://github.com/pysam-developers/pysam/issues/630

Reply:
I guess this cross-mention does the trick, then.